### PR TITLE
Clarify the behavior when a property/method does not exist

### PR DESCRIPTION
**Pull request by fabpot**
Alternative to #3964

### CODE REVIEW

**Comment by miqrogroove** [comment]
Seems a bit wordy.  Another alternative is to continue the "if not, and if" refrain.

"If not, and if strict_variables is disabled, returns null..."

**Comment by fabpot** [comment]
> Seems a bit wordy. Another alternative is to continue the "if not, and if" refrain.
> 
> "If not, and if strict_variables is disabled, returns null..."

Indeed, that's a great suggestion, updated accordingly.
